mmdet/models/roi_heads/mask_heads/fcn_mask_head.py

```diff
@@ -9,6 +9,8 @@
 
 from mmdet.core import mask_target
 from mmdet.models.builder import HEADS, build_loss
+from mmcv.ops.carafe import CARAFEPack
```

Comment:
Is it used?

Reply:
No. Removed.

Reply:
For all that it seems to be used. Please return import as it's done in upstream.

Reply:
There was double import of `CARAFEPack` and one was deleted.